### DISCUSSION TITLE
Webfollower Beonlabs stop work

### DISCUSSION THREAD

**Discussion by ibnbr** [NightscoutFoundation/xDrip#3549] [Bug reports] [answered]
The Web Follower stopped working after Libreview maintenance.

System Status message:
Last Error: Failed to get any response

**Comment by Navid200** [comment, marked as answer]
Does this help?
https://navid200.github.io/xDrip/docs/Libre3_Follow.html

**Reply by ibnbr** [reply]
Yes, thank you
